### PR TITLE
bradl3yC - 6008 - edit address messaging and focus

### PR DESCRIPTION
**Pull request by bradl3yC**
## Description
[x] - fix verbiage for scenario where you cannot override and there are no suggestions
[x] - remove last anchor and replace with button 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/user/profile/utilities/index.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ export const getValidationMessageKey = (
     !containsBadUnitNumber
   ) {
     return validationKey
-      ? ADDRESS_VALIDATION_TYPES.SHOW_SUGGESTIONS_OVERRIDE
+      ? ADDRESS_VALIDATION_TYPES.SHOW_SUGGESTIONS_NO_CONFIRMED_OVERRIDE
```
Comment (erikphansen) [inline src/platform/user/profile/utilities/index.js]: would you be so kind as to create a ticket to add unit tests for this function?

**Review by erikphansen — APPROVED** [review]
man this is getting ridiculous

File: src/platform/user/profile/utilities/index.js (hunk 1 above)
Comment (bradl3yC) [reply src/platform/user/profile/utilities/index.js]: @erikphansen we can add that to the e2e test coverage ticket

**Comment by bradl3yC** [comment]
@department-of-veterans-affairs/frontend-review-group ready for review 👍

**Review by bkjohnson — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**